client/landing/stepper/declarative-flow/internals/steps-repository/design-setup/unified-design-picker.tsx

```diff
@@ -372,6 +373,31 @@ const UnifiedDesignPickerStep: Step = ( { navigation, flow } ) => {
 		/>
 	);
 
+	const blankCanvasContent = isEnabled( 'signup/design-picker-build-your-own' ) ? (
```

Comment:
To be consistent with the className `blank-canvas-cta-wrapper`, we can use the name `blankCanvasCta` instead of `blankCanvasContent`.